### DISCUSSION TITLE
Is yubikey still not supported?

### DISCUSSION THREAD

**Discussion by BulatSaif** [FiloSottile/age#547] [Q&A]
Age project still has a warning that SSH keys held on YubiKeys can't be used to decrypt files:
https://github.com/FiloSottile/age?tab=readme-ov-file#encrypting-to-a-github-user

> #### Encrypting to a GitHub user
> Combining SSH key support and `-R`, you can easily encrypt a file to the SSH keys listed on a GitHub profile.
>
>```
>$ curl https://github.com/benjojo.keys | age -R - example.jpg > example.jpg.age
>```
>Keep in mind that people might not protect SSH keys long-term, since they are revokable when used only for authentication, and that SSH keys held on YubiKeys can't be used to decrypt files.

However, I see two projects that may solve this issue:  
- https://github.com/str4d/age-plugin-yubikey 
- https://github.com/FiloSottile/yubikey-agent  

Unfortunately, they lack information on ever how to use the existing SSH key stored in Yubikeys for age or how to connect yubikey-agent to age.

**Comment by DanNixon** [comment]
For the case of using an SSH key on a YubiKey for decryption, I believe not.

`age-plugin-yubikey` allows an age key to reside on a YubiKey. This is a key specifically for age, not an SSH key that might already exist.

`yubikey-agent` is an SSH agent that uses a YubiKey to store the private key. ([partially related question](https://github.com/FiloSottile/age/discussions/532))
